bundles/org.openhab.binding.fmiweather/OSGI-INF/org.openhab.binding.fmiweather.internal.HandlerFactory.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These OSGI-INF XML files should not be committed. This directory can be removed under the new build.

Reply:
Good observation. Resolved in 0e06b65